mpt_trie/src/trie_hashing.rs

```diff
@@ -232,7 +232,7 @@ mod tests {
         let mut trie = HashedPartialTrie::new(Node::Empty);
 
         entries.map(move |(k, v)| {
-            trie.insert(k, v);
+            assert!(trie.insert(k, v).is_ok());
```

Comment:
What's the benefit of the `assert` here?

Reply:
not sure how to answer that

Reply:
now understood what you meant